### DISCUSSION TITLE
Project in directory path Livewire not working on live server

### DISCUSSION THREAD

**Discussion by royallife88** [filamentphp/filament#9432] [Help]
Filament is working perfectly on htttps://mydomain.com . When i move the project code into new directory "myfolder" that's not working .
What i tried
.evn updated with new url  
`APP_URL = htttps://mydomain.com/myfolder   ASSET_URL = htttps://mydomain.com/myfolder`
Login loading fine but unable to login, Form submission is not working. What I noticed that's Livewire is not working. In console network tab livewire.js gives 404. livewire.js file still pointing to main domain url https://mydomain.com/livewire/livewire.js
How I can solve this problem. 
I also tried
` Livewire::setScriptRoute(function ($handle) {
            return Route::get('/myfolder/livewire/livewire.js', $handle);
        });
        Livewire::setUpdateRoute(function ($handle) {
            return Route::post('/myfolder/livewire/update', $handle);
        });`
Above code give below error
`Unable to prepare route [myfolder/livewire/update] for serialization. Another route has already been assigned name [livewire.update].`
 
 Also tried set asset_url in config/livewire.php 
 `'asset_url' => env('ASSET_URL', null)`
 But no success. Anyone can provide on this issue ASAP. Thanks

**Comment by VincvanAs** [comment]
The asset url on both could just be null. Maybe it has something to do with your .htaccess config?
